src/com/github/kornilova203/matlab/inspections/MatlabUnnecessaryControlExprVisitor.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.kornilova203.matlab.inspections
+
+import com.github.kornilova203.matlab.psi.*
+import com.intellij.codeInspection.LocalQuickFixOnPsiElement
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.TokenType
+import com.intellij.psi.tree.IElementType
+import com.intellij.psi.util.elementType
+
+open class MatlabUnnecessaryControlExprVisitor(private val holder: ProblemsHolder,
+                                               val type: IElementType,
+                                               val exprName: String) : MatlabVisitor() {
+    fun findUnnecessaryExpr(scope: PsiElement?) {
+        if (scope == null) return
+        var lastChild = scope.lastChild
+        while (lastChild != null && lastChild.elementType in setOf(TokenType.WHITE_SPACE, MatlabTypes.NEWLINE, MatlabTypes.SEMICOLON, MatlabTypes.FUNCTION_DECLARATION)) {
```

Comment:
`PsiTreeUtil.skipSiblingsBackward`

Reply:
The arguments in PsiTreeUtil.skipSiblingsBackward must be classes. Is there some classes for tokens like semicolon?

Reply:
Semicolon is just a `PsiElement` so seems like `skipSiblingsBackward` doesn't work in this case